docs/developer/using_pandas_on_python.rst

```diff
@@ -0,0 +1,37 @@
+pandas on Python
+================
+
+This section describes usage related documents for the pandas on Python component of Modin.
+
+Modin uses pandas as a primary memory format of the underlying partitions and optimizes queries
+ingested from the API layer in a specific way to this format. Thus, there is no need to care of choosing it
```

Comment:
```suggestion
from the API layer in a specific way to this format. Since it is a default, you do not need to specify
the pandas memory format, but we show how to explicitly set it below.
```